src/components/MenuButton/MenuButton.tsx

```diff
@@ -109,6 +119,7 @@ export default class MenuButton extends React.Component<MenuButtonProps, {}> {
       onClick: item.onClick,
       disabled: item.isDisabled,
       href: item.href,
+      target: item.newWindow ? '_blank' : undefined,
```

Comment:
can we also pass `rel="nofollow noopener noreferrer "` here to prevent tab hijacking attacks?
https://github.com/Microsoft/YamUI/blob/master/src/components/NavigationLink/NavigationLink.tsx#L37

Reply:
Doesn't look like Fabric supports this, we'd need to add the prop in their repo. They create the anchor tag for us

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/issues/4201